x-pack/plugins/alerting/server/task_runner/fixtures.ts

```diff
@@ -0,0 +1,383 @@
+      },
+    },
+    {
+      group: RecoveredActionGroup.id,
+      id: '2',
+      actionTypeId: 'action',
+      params: {
+        isResolved: true,
+      },
+    },
+  ],
+  executionStatus: {
+    status: 'unknown',
+    lastExecutionDate: new Date('2020-08-20T19:23:38Z'),
+  },
+  monitoring: getDefaultRuleMonitoring(),
+};
+
+export const mockTaskInstance = () => ({
+  id: '',
+  attempts: 0,
+  status: TaskStatus.Running,
+  version: '123',
+  runAt: new Date(),
+  schedule: { interval: '10s' },
+  scheduledAt: new Date(),
+  startedAt: new Date(),
+  retryAt: new Date(Date.now() + 5 * 60 * 1000),
+  state: {},
+  taskType: 'alerting:test',
+  params: {
+    alertId: RULE_ID,
+  },
+  ownerId: null,
+});
+
+export const generateAlertSO = (id: string) => ({
+  id,
+  rel: 'primary',
+  type: 'alert',
+  type_id: RULE_TYPE_ID,
+});
+
+export const generateActionSO = (id: string) => ({
+  id,
+  namespace: undefined,
+  type: 'action',
+  type_id: 'action',
+});
+
+export const generateEventLog = ({
+  action,
+  task,
+  duration,
+  start,
+  end,
+  outcome,
+  reason,
+  instanceId,
+  actionSubgroup,
+  actionGroupId,
+  status,
+  numberOfTriggeredActions,
+  savedObjects = [generateAlertSO('1')],
+}: GeneratorParams = {}) => ({
+  ...(status === 'error' && {
+    error: {
+      message: generateErrorMessage(String(reason)),
+    },
+  }),
+  event: {
+    action,
+    ...(!isNil(duration) && { duration }),
+    ...(start && { start }),
+    ...(end && { end }),
+    ...(outcome && { outcome }),
+    ...(reason && { reason }),
+    category: ['alerts'],
+    kind: 'alert',
+  },
+  kibana: {
+    alert: {
+      rule: {
+        execution: {
+          uuid: '5f6aa57d-3e22-484e-bae8-cbed868f4d28',
+          ...(!isNil(numberOfTriggeredActions) && {
+            metrics: {
+              number_of_triggered_actions: numberOfTriggeredActions,
+            },
+          }),
+        },
+      },
+    },
+    ...((actionSubgroup || actionGroupId || instanceId || status) && {
+      alerting: {
+        ...(actionSubgroup && { action_subgroup: actionSubgroup }),
+        ...(actionGroupId && { action_group_id: actionGroupId }),
+        ...(instanceId && { instance_id: instanceId }),
+        ...(status && { status }),
+      },
+    }),
+    saved_objects: savedObjects,
+    ...(task && {
+      task: {
+        schedule_delay: 0,
+        scheduled: '1970-01-01T00:00:00.000Z',
+      },
+    }),
+  },
+  message: generateMessage({ action, instanceId, actionGroupId, actionSubgroup, reason, status }),
+  rule: {
+    category: 'test',
+    id: '1',
+    license: 'basic',
+    ...(hasRuleName({ action, status }) && { name: RULE_NAME }),
+    ruleset: 'alerts',
+  },
+});
+
+const generateMessage = ({
+  action,
+  instanceId,
+  actionGroupId,
+  actionSubgroup,
+  reason,
+  status,
+}: GeneratorParams) => {
+  if (action === Actions.EXECUTE_START) {
+    return `rule execution start: "${mockTaskInstance().params.alertId}"`;
```

Comment:
As a next step these event and debugger log messages could be moved to their own `messages` file.
So, both the task_runner and its unit tests can use them from that shared source.